tests/models/mnist.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
what about name it `datasets.py` and move also `TestingMNIST` here so all datasets are in one place

Reply:
good idea